fix shared-mime-info build race condition

The definition of install-data-hook in Makefile.am leads
to multiple, overlapping, executions of install-binPROGRAMS
target.  We modify the definition to avoid that.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>

Index: shared-mime-info-1.9/Makefile.am
===================================================================
--- shared-mime-info-1.9.orig/Makefile.am
+++ shared-mime-info-1.9/Makefile.am
@@ -50,7 +50,9 @@ else
   update_mime_database="$(top_builddir)/update-mime-database$(EXEEXT)"
 endif
 
-install-data-hook: install-binPROGRAMS
+# do NOT make this dependent on anything!
+#
+install-data-hook:
 if ENABLE_UPDATE_MIMEDB
 	$(update_mime_database) -V "$(DESTDIR)$(datadir)/mime"
 endif
